packages/smooth_app/lib/generic_lib/dialogs/smooth_alert_dialog.dart

```diff
@@ -275,16 +290,23 @@ class SmoothActionButtonsBar extends StatelessWidget {
       order ?? SmoothButtonsBarOrder.auto,
       positiveAction: positiveAction,
       negativeAction: negativeAction,
+      neutralAction: neutralAction,
     )!;
 
     if (buttonsAxis == Axis.horizontal) {
-      // With two buttons, inject a small space between them
-      if (actions.length == 2) {
+      // With multiple buttons, inject a small space between them
+      if (actions.length > 1) {
         actions.insert(
           1,
           const SizedBox(width: VERY_SMALL_SPACE),
         );
       }
+      if (actions.length == 3) {
+        actions.insert(
+          3,
+          const SizedBox(width: VERY_SMALL_SPACE),
+        );
+      }
```

Comment:
Not easy to understand the code with your magic numbers (`3`), and I'm afraid your code doesn't work as intended anyway (e.g. if the length is 3 it's only in the case where we have 2 buttons and you added a space).

If we have 2 or 3 buttons, we want to add a space between them.
I suggest this - including the comments:
```suggestion
      // With multiple buttons, inject a small space between them
      if (actions.length > 1) {
        if (actions.length > 2) {
          // space injected before 3rd item
          actions.insert(
            2,
            const SizedBox(width: VERY_SMALL_SPACE),
          );
        }
        // space injected before 2nd item
        actions.insert(
          1,
          const SizedBox(width: VERY_SMALL_SPACE),
        );
      }
```

We'll probably never have 3 buttons _horizontally_, but that's another story.

Reply:
I missed that in a hurry. I have made commit with your suggested change.